src/modules/hip/kernel/tensor_stddev.hpp

```diff
@@ -0,0 +1,893 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+#include "reduction.hpp"
+
+// ----------------------- Helper Functions --------------------------
+__device__ void stddev_hip_compute(uchar *srcPtr, float src, float *dst) { *dst = src; }
+
+__device__ void stddev_hip_compute(float *srcPtr, float src, float *dst) { *dst = src * 255; }
+
```

Comment:
You could remove these blank lines and add a blank line after the separator instead.